src/components/CourseHeader.js

```diff
@@ -52,69 +36,32 @@ export default function CourseHeader(props) {
   const classes = useStyles()
   const { course } = props
 
-  if (Object.is(course.waitlist, '0')) {
-    return (
-      <CardHeader
-        className={classes.classHeader}
-        title={
-          <Typography
-            variant='subtitle1'
-            tabIndex='0'
-            className={classes.courseTitle}
-          >
-            {course.courseTitle}
-          </Typography>
-        }
-        subheader={
-          <div className={classes.classHeaderSpanDiv}>
-            <span tabIndex='0' className={classes.courseInfo}>
-              {course.subjectCode +
-                '-' +
-                course.subjectNumber +
-                '-' +
-                course.section +
-                '-' +
-                course.crn}
-            </span>
-            <span tabIndex='0' className={classes.courseInfo}>
-              Credits: {course.credit}
-            </span>
-          </div>
-        }
-      />
-    )
-  } else {
-    return (
-      <CardHeader
-        className={classes.classHeaderWaitList}
-        title={
-          <Typography tabIndex='0' className={classes.courseTitle}>
-            {course.courseTitle}
-          </Typography>
-        }
-        key={course.crn + 0 + 3}
-        subheader={
-          <div className={classes.classHeaderSpanDiv}>
-            <span tabIndex='0' className={classes.courseInfo}>
-              {course.subjectCode +
-                '-' +
-                course.subjectNumber +
-                '-' +
-                course.section +
-                '-' +
-                course.crn}
-            </span>
-            <div className={classes.subHeaderDiv}>
+  return (
+    <CardHeader
+      className={course.waitlist === '0' ? classes.classHeader : classes.classHeaderWaitList}
```

Comment:
Very nice way of handling the repetitive waitlist code

Reply:
Thank you!